example_data/sample_bundle.json

```diff
@@ -248,8 +289,8 @@
             "valueCodeableConcept": {
               "coding": [
                 {
-                  "system": "http://www.ncbi.nlm.nih.gov/refseq",
-                  "code": "NM_001350626.1"
+                  "system": "https://www.ncbi.nlm.nih.gov/refseq",
```

Comment:
we've changed this to `https:`, just highlighting to check that this was intentional